test/test_matmul_cuda.py

```diff
@@ -69,9 +69,10 @@ def test_cublas_addmm(self, size: int, dtype: torch.dtype):
         n, m, p = (size + 1, size, size + 2)
         # Disable reduced precision reductions in BFloat16 to bypass some kernels
         # which fail the threshold check
-        orig = torch.backends.cuda.matmul.allow_bf16_reduced_precision_reduction
-        if dtype == torch.bfloat16 and torch.cuda.get_device_capability() >= (8, 6):
-            torch.backends.cuda.matmul.allow_bf16_reduced_precision_reduction = False
```

Comment:
Q: Why we no longer need to guard it behind sm86?

Reply:
I think the goal is to test both reduced precision reductions on and off on all architectures it might effect, which includes e.g., sm80.